### DISCUSSION TITLE
Request & Reply error

### DISCUSSION THREAD

**Discussion by einsidhe** [open-webui/open-webui#4200] [General]
# Bug Report

## Description

Hi there.
Thank you for this beautiful project.
There is as problem with latest Open Web UI version (0.3.10).  
It drops chat, i.e. I type request, press Enter, and chat disappears.
If I reboot webpage, the following error occurs:

![01](https://github.com/user-attachments/assets/4c17252d-20af-4ca2-af8a-226c6ad7367f)

Previous versions worked fine,
Will highly appreciate if you can fix it.

## Environment

Open WebUI Version - 0.1.3
Ollama Version - 0.2.7 
Operating System - Debian 12 
Browser - Firefox 128.0.3

## Installation Method

docker

**Comment by justinh-rahb** [comment]
- #4158
